release/models/aft/openconfig-aft-common.yang

```diff
@@ -663,7 +678,11 @@ submodule openconfig-aft-common {
         "The weight applied to the next-hop within the group. Traffic
         is balanced across the next-hops within the group in the
         proportion of weight/(sum of weights of the next-hops within
-        the next-hop group).";
+        the next-hop group).
+
+        This is the programmed weight as supplied by the higher level
+        protocol or agent; it is not the necessarily the weight
+        used in the datapath programming of the NHG.";
```

Comment:
It would be useful to clarify here why this might diverge -- i.e., if we are calculating how traffic is being split, I _would_ expect that this represents the weight that is used in the data plane -- otherwise the existing leaf is not correct.

Is the case that you are handling here the one where there is some quantisation further down the programming of the forwarding plane?

Reply:
Yes, if you consider that weights programmed by gRIBI have a uint64 range and weights signaled in the BGP link bandwidth community have mantissa-exponent floating point range, it shouldn't be assumed that the full range of these weights are used by the datapath.

If this leaf is supposed to represent the quantized weight used by the datapath then this should be specified. I personally think there is value seeing both raw/signaled and normalized weights for each next-hop.

Reply:
ACK -- I would just specify in this description why the weights might diverge here. I'm open to adding something that gives us the signalled and normalised values.